auslib/db.py

```diff
@@ -1547,12 +1548,15 @@ def localeExists(self, name, platform, locale, transaction=None):
 
     def delete(self, where, changed_by, old_data_version, transaction=None, dryrun=False):
         names = []
+        engine = super(Releases, self).getEngine()
+        connection = engine.connect()
+        connection.execute("PRAGMA foreign_keys=ON")
+        connection.close()
```

Comment:
I think this is in the wrong place. This PRAGMA statement is sqlite-specific, and should only be used in the tests. If possible, please put it in the setUp() method of TestReleases. If it's not possible to put it there, it should probably go in https://github.com/mozilla/balrog/blob/7bae002dd0f178b37e1e8d73f77eafca0b3e79bd/auslib/db.py#L1912, and only run when the dialect is "sqlite".